File: src/main/java/com/SimplePrograms/ExchangedValues.java
package com.SimplePrograms;

/**
 * Example of exchanging the values of two variables without using the third
 * 
 * @author VitasSalvantes
 * @version 2.0
 */
public class ExchangedValues {

    /** Method that launches the program */
    public static void main(String[] args) {
        int a = 1, b = 2;
        System.out.println(a + " and " + b);
        a += b;
        b = a - b;
        a -= b;
        System.out.println(a + " and " + b);
    }
}

File: src/main/kotlin/com/simpleprograms/ExchangedValues.kt
package com.simpleprograms

/**
 * Example of exchanging the values of two variables without using the third
 * 
 * @author VitasSalvantes
 * @version 1.0
 */
fun  main() {
    var a = 1
    var b = 2
    println("$a and $b")
    a += b
    b = a - b
    a -= b
    println("$a and $b")
}